routes/shopifyApiProxy.js

```diff
@@ -1,4 +1,4 @@
-const querystring = require('querystring');
+const qs = require('qs');
```

Comment:
A bit nicer to still call this `querystring` since it's much more readable than an acronym.

Reply:
Sure no problem. I will also start looking at creating a test for this.

Reply:
Amazing, thank you :)